releasenotes/notes/56048.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+- 53578
+releaseNotes:
+- |
+  **Added** support customizing Ambient Enablement Labels.
```

Comment:
I am not entirely sure someone not deeply familiar with this will understand what it means. 

Do we have any docs explaining ? Maybe a link would help, or few more words on what is getting enabled and how. 

To be honest I am not a big fan of any label selectors that span multiple namespaces - it can become a big mess to maintain and secure, and I've seen plenty of hard to debug problems and security problems caused by anything that crosses boundaries - but it's hard to push back since we (ab)use this pattern in too many places and it's 'easy'. Should be fine if users who value security just don't use it.

Reply:
I completely agree that this should get some serious dcoumentation love, and also that directly modifying these values is likely for advanced users or service providers.  In a future PR, I hope to add some templates such as "opt in, opt out, etc".